src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingOptions.cs

```diff
@@ -32,6 +32,16 @@ public static RazorFormattingOptions From(FormattingOptions options, bool codeBl
         };
     }
 
+    public static RazorFormattingOptions From(RoslynFormattingOptions options, bool codeBlockBraceOnNextLine)
+    {
+        return new RazorFormattingOptions()
```

Comment:
>RazorFormattingOptions [](http://example.com/codeflow?start=19&length=22)

Nit: I believe type name is unnecessary? This file is a bit messy type name in some places but just "new" on others.

Reply:
Will have a look and clean it up when I bump Roslyn. Appreciate the review.